crates/telio-model/src/features.rs

```diff
@@ -180,6 +183,10 @@ impl FeatureNurse {
     fn default_enable_nat_traversal_conn_data() -> bool {
         true
     }
+
+    fn default_state_duration_cap() -> u64 {
```

Comment:
If it's not possible to use Duration type to describe this duration, it would help a lot if the measurement units were appended to this function name or at least in a comment. e.g. default_state_duration_cap_s(). Especially if the value changed in the future and the measurement units would become less obvious for this value.